lib/sprockets/sassc_processor.rb

```diff
@@ -0,0 +1,77 @@
+require 'sprockets/sassc_importer'
+require 'sprockets/sass_processor'
+
+module Sprockets
+  class SasscProcessor < SassProcessor
+    def initialize(options = {}, &block)
+      @cache_version = options[:cache_version]
+      @cache_key = "#{self.class.name}:#{VERSION}:#{Autoload::SassC::VERSION}:#{@cache_version}".freeze
+      @importer_class = options[:importer] || SasscImporter
+      @functions = Module.new do
+        include SassProcessor::Functions
+        include Functions
+        include options[:functions] if options[:functions]
+        class_eval(&block) if block_given?
+      end
+    end
+
+    def call(input)
+      context = input[:environment].context_class.new(input)
+
+      options = {
+        filename: input[:filename],
+        syntax: self.class.syntax,
+        load_paths:  input[:environment].paths,
+        importer: @importer_class,
+        sprockets: {
+          context: context,
+          environment: input[:environment],
+          dependencies: context.metadata[:dependencies]
+        }
+      }
+
+      engine = Autoload::SassC::Engine.new(input[:data], options)
+
+      css = Utils.module_include(Autoload::SassC::Script::Functions, @functions) do
+        engine.render
+      end
+
+      # Track all imported files
+      sass_dependencies = Set.new([input[:filename]])
+      engine.dependencies.map do |dependency|
```

Comment:
FYI - `engine#dependencies` does not appear to work properly, and is probably not required anyway since all dependencies are tracked in the importer.